src/evm/simulation.rs

```diff
@@ -114,24 +117,31 @@ where
             ..Default::default()
         };
 
-        let mut cfg_env: CfgEnv<SpecId> = CfgEnv::default();
+        let mut cfg_env: CfgEnv<SpecId> = CfgEnv::new_with_spec(SpecId::PRAGUE);
```

Comment:
What does this do

Reply:
it changes the version of the EVM we are using (related to the upgrades) 
but this change here does literally nothing 😅 sorry. because the default one is Prague